frontend/src/app/search/highlight-selector.component.ts

```diff
@@ -2,7 +2,7 @@ import { Component, Input, OnChanges, OnDestroy, SimpleChanges } from '@angular/
 import { QueryModel } from '../models';
 
 
-const HIGHLIGHT = 200;
+const HIGHLIGHT = 100;
```

Comment:
Isn't this an unreachable value? Why is it the default? Do you need a default value at all?

Reply:
removed it!